src/components/WalletButton.tsx

```diff
@@ -32,11 +32,19 @@ const WalletButtonWrapper = ({
   return (
     <Button
       variant="outlined"
-      sx={{padding: "10px 25px", backgroundColor: theme.palette.mode === "dark" ? "#1B1F1E" : "white",color: theme.palette.mode === "dark" ? "#2ED8A7" : "#121615"}}
+      sx={{
+        padding: "10px 25px",
+        backgroundColor: theme.palette.mode === "dark" ? "#1B1F1E" : "white",
```

Comment:
Can it use colors from the global color palette?

Reply:
I'll leave this change to @msavir for #78 to avoid merge conflicts.